src/controllers/rowColumnOperation.js

```diff
@@ -1629,6 +1629,9 @@ export function rowColumnOperationInitial(){
                         if(getObjType(d[r][c]) == "object"){
                             delete d[r][c]["m"];
                             delete d[r][c]["v"];
+                            if(d[r][c]["ct"] != null) {
+                                delete d[r][c]["ct"]["s"]
+                            }
```

Comment:
```suggestion
                            if(d[r][c]["ct"] != null) {
                                delete d[r][c]["ct"]["s"]
                            }
```
这里需要把d[r][c]["ct"].t == 'inlineStr' ,inlineStr重置为g，因为单元格为行内样式的话，会设置t为inlineStr，这里不重置会有隐患

Reply:
```js
if(d[r][c]["ct"] != null && d[r][c]["ct"].t == 'inlineStr'){
  delete d[r][c]["ct"];
 }
```
刚看这个问题已经被别的同学修复了，我给这个删了